TAF_EasyRest/TestFramework/PageComponents/AdministratorPanelComponents/AcceptedCardPageComponent.cs

```diff
@@ -0,0 +1,35 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class AcceptedCardPageComponent
+    {
+        IWebDriver driver { get; }
```

Comment:
please specify the access modifier explicitly

Reply:
done